evap/evaluation/models.py

```diff
@@ -984,12 +984,20 @@ def textanswer_review_state(self):
     def ratinganswer_counters(self):
         return RatingAnswerCounter.objects.filter(contribution__evaluation=self)
 
+    @property
+    def all_participants_are_external(self):
+        for participant in self.participants.all():
+            if not participant.is_external:
+                return False
+        return True
```

Comment:
```suggestion
        return all(participant.is_external for participant in self.participants.all())
```
or at this point we might just inline the function?